examples/tensorflow-adversarial-patches/Makefile

```diff
@@ -61,16 +62,22 @@ TAR = tar
 CODE_SRC_FILES := $(wildcard $(EXAMPLE_SRC_DIR)/*.py)
 CODE_WORKFLOW_FILE = MLproject
 
+PLUGINS_CUSTOM_DIR := $(EXAMPLE_CUSTOM_PLUGINS_DIR)/securingai_custom
+CUSTOM_PLUGIN_EVALUATION_SRC_FILES := $(wildcard $(PLUGINS_CUSTOM_DIR)/custom_patch_plugins/*.py)
+
 JUPYTER_INSTALL_PACKAGES_SCRIPT = $(EXAMPLE_JUPYTER_DIR)/before-notebook.d/install-packages.sh
 
 MAKEFILE_FILE = Makefile
+
+CUSTOM_TASK_PLUGINS_EVALUATION_TARBALL_FILE = custom-plugins-evaluation.tar.gz
```

Comment:
I don't want to get too caught up about the variable names (i.e. the ones in all caps) in the Makefile, but the tar file itself should probably have a different name, for example `custom-plugins-custom-patch-plugins.tar.gz`, otherwise users might get confused if they made a mistake.